code/game/machinery/computer/arcade.dm

```diff
@@ -584,6 +610,108 @@
 	else if(href_list["eventclose"]) //end an event
 		event = null
 
+	else if(href_list["killcrew"]) //shoot a crewmember
+		var/sheriff = remove_crewmember() //I shot the sheriff
+		playsound(loc,'sound/weapons/Gunshot.ogg', 100, 1)
+
+		if(lings_aboard && prob(40*lings_aboard)) //if there are 2 lings you're twice as likely to get one, obviously
+			lings_aboard = max(0,--lings_aboard)
+
+		if(settlers.len == 0 || alive == 0)
+			say("The last crewmember [sheriff], shot themselves, GAME OVER!")
+			if(emagged)
+				usr.death(0)
+				emagged = 0
+			gameover = 1
+			event = null
+		else if(emagged)
+			if(usr.name == sheriff)
+				say("The crew of the ship chose to kill [usr.name]!")
+				usr.death(0)
+
+		if(event == ORION_TRAIL_LING) //only ends the ORION_TRAIL_LING event, since you can do this action in multiple places
+			event = null
+
+	//Spaceport specific interactions
+	//they get a header because most of them don't reset event (because it's a shop, you leave when you want to)
+	//they also call event() again, to regen the eventdata, which is kind of odd but necessary
+	else if(href_list["buycrew"]) //buy a crewmember
+		var/bought = add_crewmember()
+		last_spaceport_action = "You hired [bought] as a new crewmember."
+		fuel -= 10
+		food -= 10
+		event()
+
+	else if(href_list["sellcrew"]) //sell a crewmember
+		var/sold = remove_crewmember()
+		last_spaceport_action = "You sold your crewmember, [sold]!"

```

Comment:
Well, you lose strength for the combat events, and come closer to death.
Same repercussions as just shooting the guy in the head, except you get paid.

also I was so engrained in the dangerously shitty world of Orion Trail that slavery was just natural :P